src/taskItem.tsx

```diff
@@ -39,7 +41,7 @@ function TaskItem({
 export const listItem: NodeRenderer<ListItem & { checked?: boolean }> = ({
   node
 }) => {
-  if (node.checked == null) {
+  if (node.checked === null) {
```

Comment:
@agoose77 this is actually different logic! Causing the bug in #220.

`== null` checks for null and undefined.
`=== null` is a strict check for null.

We could also move to `!node.checked` here which is even more expansive.